pointerlock/interfaces.window.js

```diff
@@ -0,0 +1,24 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/pointerlock/
+
+promise_test(async () => {
+  const dom = await fetch('/interfaces/dom.idl').then(r => r.text());
+  const uievents = await fetch('/interfaces/uievents.idl').then(r => r.text());
+  const idl = await fetch('/interfaces/pointerlock.idl').then(r => r.text());
+
+  var idl_array = new IdlArray();
+  idl_array.add_untested_idls(dom);
+  idl_array.add_untested_idls(uievents);
+  idl_array.add_idls(idl);
+  
+  idl_array.add_objects({
+    Document: ["window.document"],
+    Element: ["window.document.documentElement"],
+    MouseEvent: ["new MouseEvent('mousemove')", "new MouseEvent('pointerlockchange')"]
```

Comment:
This was preexisting, but I think that testing two MouseEvent instances that only differ by the *value* of the type attribute gets no extra coverage. Optional cleanup in other words.